### ISSUE TITLE
feat(monkey-kernel): wire short-side execution end-to-end (long+short symmetry, lockout, override balance preserved)

### ISSUE THREAD

**Issue by GaryOcean428** [GaryOcean428/poloniex-trading-platform#581]
## Goal

Wire short-side execution end-to-end through `apps/api/src/services/monkey/loop.ts`. Currently the kernel only emits `enter_long`. Production logs from the entire post-Phase-B window show `side: long` on every entry. After #579 confirms the ML can produce SELL, this issue makes the execution path follow.

### 1. Direction decision in detect_mode (Python side)

`detect_mode()` already takes the ML signal as input. Add to its return:

```python
return {
    ...
    "direction": derive_direction(ml_signal, basin_dir, lane).value,  # "long" | "short" | "flat"
    ...
}
```

Where `derive_direction`:

```python
def derive_direction(ml_signal: str, basin_direction: float, lane: Lane) -> Side:
    if lane == Lane.OBSERVE:
        return Side.FLAT
    if ml_signal == "HOLD":
        return Side.FLAT
    if ml_signal == "BUY":
        return Side.LONG
    if ml_signal == "SELL":
        return Side.SHORT
    return Side.FLAT  # safety fallback
```

Per-lane bias modifiers possible later (e.g. SCALP can mean-revert), but v1 is straight ML-signal passthrough.

### 2. Loop wiring (TS side)

In `apps/api/src/services/monkey/loop.ts`:

- Find the existing `enter_long` action emitter
- Add `enter_short` parallel path
- Order placement: `side="short"` translates to a SELL order on Poloniex perps (futures: `side=SELL` opens a short position)
- TP/SL inversion: for a short, profit = entry > mark, loss = entry < mark. Risk kernel rules must respect this
- Trailing harvest: existing logic computes pnlFrac as `(entry - mark)/entry` for shorts vs `(mark - entry)/entry` for longs

### 3. Override balance (CRITICAL)

The kernel has an existing `override` field that previously created a **permanent SHORT bias** before the basin saturation fix (commit `589c7756`). After the fix, override is balanced. This issue MUST NOT regress that — verify with a test that `override` distribution stays balanced under live signals after wiring shorts.

### 4. Risk kernel parity

`ml-worker/src/trading/risk_kernel.py` — confirm SL/TP math is symmetric for shorts:

```python
# long: sl_price = entry * (1 - sl_threshold)
# short: sl_price = entry * (1 + sl_threshold)
# long: tp_price = entry * (1 + tp_threshold)
# short: tp_price = entry * (1 - tp_threshold)
```

Test: `tests/trading/test_risk_kernel_parity.py` already exists. Add short-side test cases.

## Hard constraints

- **No permanent short bias**. After #574 dedup + this PR, the kernel must not slide back into "everything is short" via override pollution
- **Symmetric SL/TP math for shorts**. A 70 bps SL on a long means `mark < entry * 0.993`; on a short it means `mark > entry * 1.007`
- **Lockout**: the kernel cannot hold both long and short on the same `(symbol, lane)` simultaneously. Long position must be exited before short can be entered, and vice versa
- **Sovereignty preserved**. ML signal can flip direction but cannot exceed 30% slerp weight against identity basin
- `qig_purity_check.py` clean

## Tests required

- `test_short_execution.py` (Python): given mock ML signal "SELL", `detect_mode` returns `direction="short"`, `enter_short` action emits
- `test_short_pnl_math.py` (Python): SL/TP price math symmetric for short positions
- `test_lock_out.py` (Python): cannot enter short while long position open (and vice versa) on same `(symbol, lane)` pair
- `loop.test.ts`: integration — kernel decision `direction=short` results in correct order shape sent to exchange adapter (mocked)
- `test_override_balance.py`: post-wiring, `override` distribution stays within ±10% of zero over 1000 simulated ticks

## Out of scope

- Cross-symbol hedging (long ETH + short BTC for net delta exposure). Future
- Funding-rate-aware entry decisions. Future
- Direction flip without exit (some exchanges support direct flip orders). v1 always exits before flipping
- Retraining ML for short capability — handled by #579 verdict if needed

## Acceptance criteria

- [ ] `derive_direction()` exists and returns `LONG`/`SHORT`/`FLAT` correctly
- [ ] `detect_mode` return shape includes `direction`
- [ ] TS loop wires `enter_short` parallel to `enter_long`
- [ ] Risk kernel SL/TP math is symmetric (parity test passes)
- [ ] Lockout holds: same `(symbol, lane)` cannot hold both directions
- [ ] Override balance test passes — no regression to permanent-short bias
- [ ] Within 7 days post-deploy: at least one short trade fires (or #579 audit explains why ML is currently in long-only conditions)
- [ ] `qig_purity_check.py` clean

## Sequencing

Strictly blocked on #579. If #579 verdict is "C — model is long-only," this issue waits until retraining lands. If verdict is A/B/D, this issue proceeds with the appropriate fix.

🤖 Sub-issue authored by Claude Opus 4.7.

**Comment by GaryOcean428** [comment]
Resolved (implemented + live). Short-side execution is fully wired end-to-end:
- enter_short action in apps/api/src/services/monkey/loop.ts:345 (L decision agent_L_classifier)
- K and L kAction lifecycle gates handle enter_short symmetrically with enter_long (loop.ts:277-353)
- Live verified: 17 'side":"short"' lines in last 1000 Railway log entries
- HEDGE-mode posSide handling shipped in PR #676 (LiveSignal) + PR #761 (close-race coordinator handles both sides)

If specific edge cases remain (lockout, ordering, etc.) file focused follow-ups.
